### PR TITLE
ZCS-1885: Universal UI - Import/Export interface update

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Updated look and feel of Preferences => Import/Export interface as per new mockup.

https://jira.corp.synacor.com/browse/ZCS-1885

### CODE REVIEW

File: WebRoot/js/zimbraMail/share/view/ZmImportView.js [hunk 1]
```diff
@@ -154,4 +156,11 @@ ZmImportView.prototype._handleFileChange = function(file) {
 	if (type) {
 		this.set("TYPE", type);
 	}
+
+	//show uploaded file-name in custom file-upload widget
+	var fileNameWrapper = Dwt.byId(ZmImportView.ID + "_FILE_NAME", this.getControl("FORM"));
+	if(fileNameWrapper) {
```
Comment (SatyamKumarOnline) [inline WebRoot/js/zimbraMail/share/view/ZmImportView.js]: Minor indentation

File: WebRoot/skins/_base/base4/skin.properties [hunk 2]
```diff
@@ -2367,3 +2368,22 @@ PrefMailComposeRowValueWrapper                 = width:@PrefMailComposeRowValueW
...
 PrefMailComposeRowSelectWidget                 = min-width:@PrefRowMiniControlWidth@
 PrefMailComposeRowColorPickerWidget            = width:@PrefMailComposeRowColorPickerWidgetWidth@
+
+######################################
+# Preference- Import-Export interface
+######################################
+
+PrefImpExtPage                  = 
+PrefImpExpActionBtn             = display:inline-block;
+PrefImpExpChooseFolderBtn       = width:auto; position:relative; top:4px;
```
Comment (SatyamKumarOnline) [inline WebRoot/skins/_base/base4/skin.properties]: Why we going with auto value, here

**Comment by SatyamKumarOnline** [comment]
Looks good to me

File: WebRoot/skins/_base/base4/skin.properties (hunk 2 above)
Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: To override width being applied by other rule viz. `ZHasText .ZButtonTable`, which was adding width of `60px`

Thanks

File: WebRoot/js/zimbraMail/share/view/ZmImportView.js (hunk 1 above)
Comment (kalpeshhpatel) [inline WebRoot/js/zimbraMail/share/view/ZmImportView.js]: You mean space before `{`. 

If so, it's consistent with others. If you look around: https://github.com/Zimbra/zm-web-client/blob/9be691c62c51a56f0b61ffb9a28e1b706cd47ce7/WebRoot/js/zimbraMail/share/view/ZmImportView.js#L152

File: WebRoot/skins/_base/base4/skin.properties (hunk 2 above)
Comment (SatyamKumarOnline) [inline WebRoot/skins/_base/base4/skin.properties]: Ok.

**Pull request merged**